packages/vscode-extension/src/handlers.ts

```diff
@@ -303,6 +304,17 @@ async function getIsFromSample() {
   return undefined;
 }
 
+async function getIsM365() {
+  if (core) {
+    const input = getSystemInputs();
+    input.ignoreEnvInfo = true;
+    await core.getProjectConfig(input);
+
+    return core.isM365;
```

Comment:
Is it possible to use projectSettings.isM365? Then no need to update core field.

Reply:
Removed the changes in core